packages/nvidia_nat_ragas/tests/test_rag_evaluate.py

```
@@ -46,6 +46,41 @@ class ExampleModel(BaseModel):
+def fixture_atif_samples(rag_user_inputs, rag_expected_outputs, rag_generated_outputs):
+    """ATIF-native samples for testing RAG ATIF evaluator path."""
+    from nat.data_models.atif import ATIFAgentConfig
+    from nat.data_models.atif import ATIFObservation
+    from nat.data_models.atif import ATIFObservationResult
+    from nat.data_models.atif import ATIFStep
+    from nat.data_models.atif import ATIFTrajectory
+    from nat.plugins.eval.evaluator.atif_evaluator import AtifEvalSample
+
+    samples = []
+    for index, (user_input, expected_output,
+                generated_output) in enumerate(zip(rag_user_inputs, rag_expected_outputs, rag_generated_outputs)):
```

Ruff (0.15.4)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)